packages/block-library/src/list/style.scss

```diff
@@ -0,0 +1,4 @@
+ol.has-background,
+ul.has-background {
+	padding: $block-bg-padding--v $block-bg-padding--h;
```

Comment:
This padding is being overridden by [the default editor styles](https://github.com/WordPress/gutenberg/blob/c19d2d908cba695960cf8407bd0b0afc181aa657/packages/editor/src/editor-styles.scss#L103). You might want to make the selector `.block-editor-block-list__block.has-background` instead (that should also work for both types of lists)

Reply:
`block-editor-block-list__block` is a class applied to all blocks in the editor, so I don't think I want to use that selector. (Also, that class is only applied in the editor.)

The default editor style rule (which gets loaded as `.editor-styles-wrapper ol, .editor-styles-wrapper ul`) has equal specificity as `ol.has-background, ul.has-background`. The only reason why the default editor styles take precedence is because they are loaded after the block styles. That sounds like a bug to me.

Reply:
Oh, you're right, that's a general block class 🤦
I'm not 100% sure why the default styles are loaded inline. Might be worth creating an issue to see if we can change that.

Reply:
I've created #24011 to track this bug.

Reply:
That issue is now fixed. I've created #30094 to remove the now-unnecessary editor styles.